doc/modules/linear_model.rst

```diff
@@ -897,6 +897,169 @@ to warm-starting (see :term:`Glossary <warm_start>`).
     .. [9] `"Performance Evaluation of Lbfgs vs other solvers"
             <http://www.fuzihao.org/blog/2016/01/16/Comparison-of-Gradient-Descent-Stochastic-Gradient-Descent-and-L-BFGS/>`_
 
+.. _Generalized_linear_regression:
+
+Generalized Linear Regression
+=============================
+
+Generalized Linear Models (GLM) extend linear models in two ways
+[10]_. First, the predicted values :math:`\hat{y}` are linked to a linear
+combination of the input variables :math:`X` via an inverse link function
+:math:`h` as
+
+.. math::    \hat{y}(w, x) = h(x^\top w) = h(w_0 + w_1 x_1 + ... + w_p x_p).
+
+Secondly, the squared loss function is replaced by the unit deviance :math:`d`
+of a reproductive exponential dispersion model (EDM) [11]_. The minimization
+problem becomes
+
+.. math::    \min_{w} \frac{1}{2 \sum_i s_i} \sum_i s_i \cdot d(y_i, \hat{y}(w, x_i)) + \frac{\alpha}{2} ||w||_2
+
+with sample weights :math:`s`, and L2 regularization penalty :math:`\alpha`.
+The unit deviance is defined by the log of the :math:`\mathrm{EDM}(\mu, \phi)`
+likelihood as
+
+.. math::     d(y, \mu) = -2\phi\cdot
+              \left( \log p(y|\mu,\phi)
+              - \log p(y|y,\phi)\right).
+
+The following table lists some specific EDM distributions—all are Tweedie
+distributions—and some of their properties.
+
+================= ===============================  ====================================== ============================================
+Distribution       Target Domain                    Unit Variance Function :math:`v(\mu)`  Unit Deviance :math:`d(y, \mu)`
+================= ===============================  ====================================== ============================================
+Normal            :math:`y \in (-\infty, \infty)`  :math:`1`                              :math:`(y-\mu)^2`
+Poisson           :math:`y \in [0, \infty)`        :math:`\mu`                            :math:`2(y\log\frac{y}{\mu}-y+\mu)`
+Gamma             :math:`y \in (0, \infty)`        :math:`\mu^2`                          :math:`2(\log\frac{\mu}{y}+\frac{y}{\mu}-1)`
+Inverse Gaussian  :math:`y \in (0, \infty)`        :math:`\mu^3`                          :math:`\frac{(y-\mu)^2}{y\mu^2}`
+================= ===============================  ====================================== ============================================
+
+
+Usage
+-----
+
+In the following use cases, a loss different from the squared loss might be
+appropriate:
+
+  * If the target values :math:`y` are counts (non-negative integer valued) or
+    frequencies (non-negative), you might use a Poisson deviance with log-link.
```

Comment:
relative frequencies?

Reply:
I don't understand the term "relative frequencies"; i.e. relative to what?

Reply:
> I don't understand the term "relative frequencies"

Me neither ^^

But I think in English a frequency is usually a count, whereas in French it's... A (relative) frequency, aka a ratio.

@jnothman maybe?

Reply:
> But I think in English a frequency is usually a count, whereas in French it's... A (relative) frequency, aka a ratio

Are you sure? At least in physics frequency has the same meaning in English and in French as far as I know https://en.wikipedia.org/wiki/Frequency i.e. number of occurrences per unit of time, not just a count.

Reply:
Yes but https://en.wikipedia.org/wiki/Frequency_(statistics) ^^

Reply:
Hah, I didn't know about that one. So according to wikipedia for statistics, in English frequency = absolute frequency, while in French frequency = relative frequency. I think the English version makes more sense with respect to the commonly defined definition outside of statistics. 

Absolute frequency : number of occurrences during the experiment (presumably of some duration), so the units are still [occurrences / time]. It's just happens that it out case users have a different duration of observation (or exposure), so you normalize by that to compare users. That's still an absolute frequency, as far as I understand.

A relative frequency would be to normalize all those by the sum of frequencies, i.e. scale `y` by a constant factor, the norm, it shouldn't matter much but I don't see the point in this use case.

So I think this applies to both absolute and relative frequencies, but saying just "frequency" is probably more general.